tests/lmeval/test_lmeval.py

```diff
@@ -73,6 +75,12 @@ def set_up(self):
         self.quant_type = eval_config.get("quant_type")
         self.save_dir = eval_config.get("save_dir")
 
+        seed = eval_config.get("seed", None)
+        if seed is not None:
```

Comment:
nit: use `from transformers import set_seed`
https://github.com/huggingface/transformers/blob/94ae1ba5b55e79ba766582de8a199d8ccf24a021/src/transformers/trainer_utils.py#L92

Reply:
thanks for the reference. I think i will leave as is though so it remains explicit